interface-elements-for-web/articles/grid/data-summaries.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Data Summaries
+---
+# Data Summaries
+
+The grid can calculate and displays summaries, i.e., brief information about groups of rows or individual data columns. The following aggregate functions are available:
+* Average
+* Count
+* Max
+* Min
+* Sum
+
+The grid supports two types of summaries: total and group summary.
+
+## Total Summary
+A total summary calculates the value of an aggregate function over all the rows within a grid and is displayed within the footer.
+
+![eud-summaries-total](../../images/grid-summary-total.png)  
+
+## Group Summary
+A group summary calculates the value of an aggregate function over all rows within a group and is displayed in the group row or group footer.
```

Comment:
group row or group footer -> group row or footer.

Reply:
A Grid has two different elements a **footer** and a **group footer**. I think if I write "group row or footer" it can be a bit confusing.

Reply:
Ok